src/client/Client.cc

```diff
@@ -3717,14 +3738,14 @@ class C_Client_FlushComplete : public Context {
   Client *client;
   InodeRef inode;
 public:
-  C_Client_FlushComplete(Client *c, Inode *in) : client(c), inode(in) { }
+  C_Client_FlushComplete(Client *c, Inode *in) : client(c), inode(in) {}
```

Comment:
nit: unrelated change - please revert.

or, have a separate commit if it is _really_ necessary.